osu.Game/Overlays/Rankings/RankingsOverlayHeader.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Bindables;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets;
+using osu.Game.Users;
+using System.Collections.Generic;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Allocation;
+
+namespace osu.Game.Overlays.Rankings
+{
+    public class RankingsOverlayHeader : TabControlOverlayHeader<RankingsScope>
+    {
+        public readonly Bindable<RulesetInfo> Ruleset = new Bindable<RulesetInfo>();
+        public readonly Bindable<Spotlight> Spotlight = new Bindable<Spotlight>();
+        public readonly Bindable<Country> Country = new Bindable<Country>();
+
+        public IEnumerable<Spotlight> Spotlights
+        {
+            get => spotlightsContainer.Spotlights;
+            set => spotlightsContainer.Spotlights = value;
+        }
+
+        protected override ScreenTitle CreateTitle() => new RankingsTitle
+        {
+            Scope = { BindTarget = Current }
+        };
+
+        protected override Drawable CreateTitleContent() => new OverlayRulesetSelector
+        {
+            Current = Ruleset
+        };
+
+        private SpotlightsContainer spotlightsContainer;
+
+        protected override Drawable CreateContent() => new FillFlowContainer
+        {
+            RelativeSizeAxes = Axes.X,
+            AutoSizeAxes = Axes.Y,
+            Direction = FillDirection.Vertical,
+            Children = new Drawable[]
+            {
+                new CountryFilter
+                {
+                    Current = Country
+                },
+                spotlightsContainer = new SpotlightsContainer
+                {
+                    Spotlight = { BindTarget = Spotlight }
+                }
+            }
+        };
+
+        protected override void LoadComplete()
+        {
+            Current.BindValueChanged(onCurrentChanged, true);
+            base.LoadComplete();
+        }
+
+        private void onCurrentChanged(ValueChangedEvent<RankingsScope> scope) =>
+            spotlightsContainer.FadeTo(scope.NewValue == RankingsScope.Spotlights ? 1 : 0, 200, Easing.OutQuint);
```

Comment:
I think country should also be faded? I don't believe this sort of display is possible on osu-web:

![image](https://user-images.githubusercontent.com/1329837/73715433-485cf580-4757-11ea-9939-0a07b603372a.png)

Reply:
This logic happens in the overlay itself

Reply:
Hmm... Ok for now then.